src/SourceBuild/content/repo-projects/sdk.proj

```diff
@@ -20,8 +20,6 @@
          or from the Rid if supplied. -->
     <BuildArgs Condition="$(PortableBuild) != 'true'">$(BuildArgs) /p:OSName=$(TargetRid.Substring(0, $(TargetRid.IndexOf("-"))))</BuildArgs>
     <BuildArgs>$(BuildArgs) /p:PortableOSName=$(__PortableTargetOS)</BuildArgs>
-    <BuildArgs>$(BuildArgs) /p:TargetRid=$(TargetRid)</BuildArgs>
```

Comment:
do we already pass in TargetRid elsewhere or do we no longer need it?

Reply:
Yes. It's already passed in here: https://github.com/dotnet/sdk/blob/4f874f0d7f6e0fb4e39dd8f8235aa87dbe4ce186/src/SourceBuild/content/repo-projects/Directory.Build.props#L113